jdi-light-mobile-examples/src/main/java/selenium/seleniumPO/LoginForm.java

```diff
@@ -0,0 +1,18 @@
+package selenium.seleniumPO;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import selenium.entities.User;
+
+public class LoginForm {
+    @FindBy(css="form #name")      WebElement userName;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)